webpack/assets/javascripts/react_app/redux/API/APIReducer.js

```diff
@@ -18,11 +18,13 @@ const apiReducer = (state = initialState, { type, key, payload, response }) => {
           ...state[key],
           payload,
           status: PENDING,
+          isFirstRequest: !state[key],
         },
       });
     case SUCCESS:
       return state.merge({
         [key]: {
+          ...state[key],
```

Comment:
My concern is that merging old values with new values of the same API request might lead to future errors,  don't the new values enough?

Reply:
so in this case [`response, status, payload`](https://github.com/theforeman/foreman/pull/7837/files/7acf7a356bfe3d72385cf94023859feffc4cf33e#diff-4271735b1b87e411761d8e641b454d43R28-R30) will override the destructed values,

it was only added so we don't need to couple keys that doesn't change, like the `isFirstRequest`